sympy/physics/units/__init__.py

```diff
@@ -162,6 +162,7 @@
     josephson_constant,
     von_klitzing_constant,
     amu, amus, atomic_mass_unit, atomic_mass_constant,
+    me, electron_rest_mass_constant,
```

Comment:
not everything needs to end with "_constant"...I would recommend just "electron_rest_mass"